### ISSUE TITLE
[low] Chrome + Jaws: Focus lands twice on the 'Download links'.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#9145]
## Action Performed:
1. Using Chrome + Jaws , open URL staging.new.expensify.com
2. Navigate to the Setting using Tab key and Press Enter.
3. Navigate to the about using Tab key and Press Enter.
4. Navigate to the App download links using Tab key and Press Enter.
5. New screen will appear, verify the focus order and observe the issue.

## Expected Result:
Focus should lands on the each item once. Focus should land on the 'Download links' once.

## Actual Result:
Focus lands on the 'Download links' twice. When focus lands on the Download links, it announced as "android" again we press the tab key focus still appear on the same element and announce the same.

## Workaround:
Yes

## Area issue was found in:
App download links

## Failed WCAG checkpoints
2.4.3

## User impact:
Keyboard and screen reader users might be confused having extra focus on single element.

## Suggested resolution:
Ensure focus lands on the 'Download links' only once.

Refer to:
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

## Platform:

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** v1.1.66-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

https://user-images.githubusercontent.com/97600950/170037951-69665fb9-8168-4e4e-aeda-382b6ef048ce.mp4
